components/script/dom/gpucanvascontext.rs

```diff
@@ -103,14 +103,17 @@ pub struct GPUCanvasContext {
     #[ignore_malloc_size_of = "channels are hard"]
     #[no_trace]
     channel: WebGPU,
-    /// https://gpuweb.github.io/gpuweb/#dom-gpucanvascontext-canvas
+    /// <https://gpuweb.github.io/gpuweb/#dom-gpucanvascontext-canvas>
     canvas: HTMLCanvasElementOrOffscreenCanvas,
+    #[ignore_malloc_size_of = "manual writing is hard"]
+    /// <https://gpuweb.github.io/gpuweb/#dom-gpucanvascontext-configuration-slot>
+    config: RefCell<Option<GPUCanvasConfiguration>>,
```

Comment:
Is the configuration ever read other than to know whether to reconfigure the context? Why not just store a bool?